packages/components/pie-card-container/src/index.ts

```diff
@@ -38,17 +38,21 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
     @property({ type: Object })
     public aria!: AriaProps;
 
+    @property({ type: Boolean })
+    public isDraggable = false;
+
     /**
      * Renders the card as an anchor element.
      *
      * @private
      */
     private renderAnchor (): TemplateResult {
         return html`
-            <a  
+            <a
                 class="c-card-container"
                 data-test-id="pie-card-container"
                 interactionType=${this.interactionType}
+                isDraggable="${this.isDraggable}"
```

Comment:
boolean values in lit should be marked with ? otherwise, they get attached to the dom when they are false.
```suggestion
                ?isDraggable="${this.isDraggable}"
```